roles/cloud_connector/tasks/main.yml

```diff
@@ -0,0 +1,100 @@
+- name: Install rhc
+  block:
+    - name: Check if foreman-maintain installed
+      ansible.builtin.package_facts:
+        manager: auto
+
+    - name: Unlock packages
+      command: "foreman-maintain packages unlock"
+      when: "'foreman-maintain' in ansible_facts.packages"
+
+    - name: Install yggdrasil-worker-forwarder
+      package:
+        name: yggdrasil-worker-forwarder
+        state: present
+
+    - name: Install rhc
+      package:
+        name: rhc
+        state: present
+
+    - name: Lock packages
+      command: "foreman-maintain packages lock"
+      when: "'foreman-maintain' in ansible_facts.packages"
+
+- name: Create workers directory
+  file:
+    state: directory
+    path: "/etc/rhc/workers"
+    owner: root
+    group: root
+    mode: 755
+
+- name: Configure rhc-cloud-connector-worker
+  copy:
+    dest: "/etc/rhc/workers/rhc-worker-cloud-connector.toml"
+    content: |
+      directive = 'cloud-connector'
+      env = [
+        "YGG_FORWARDER_USER={{ username }}",
+        "YGG_FORWARDER_PASSWORD={{ password }}",
+        "YGG_FORWARDER_URL={{ url }}"
```

Comment:
This will change sooner than later. Yggdrasil does not allow passing ENV variables with prefix `YGG_`